users/views.py

```diff
@@ -22,4 +22,22 @@ def register(request):
 
 @login_required
 def profile(request):
-    return render(request, 'users/profile.html')
+
+    if request.method == 'POST':
+        u_form = UserUpdateForm(request.POST, instance=request.user)
+        p_form = ProfileUpdateForm(
```

Comment:
Please change the variables names from u_form and p_form to user_form and profile_form.

Reply:
Done